<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserAddDialog</class>
 <widget class="QDialog" name="UserAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Personal Details</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="FirstNameLabel">
          <property name="text">
           <string>FirstName:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="FirstNameText"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="LastNameLabel">
          <property name="text">
           <string>LastName:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="PhoneNumberLabel">
          <property name="text">
           <string>Phone Number:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="EmailLabel">
          <property name="text">
           <string>Email Address:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="IDNumberLabel">
          <property name="text">
           <string>ID Number:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="LastNameText"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="PhoneNumberText"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="EmailAddressText"/>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="IDNumberText">
          <property name="maxLength">
           <number>7</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Mailing Lists</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="ProductionCheck">
            <property name="text">
             <string>Production</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="TVCheck">
            <property name="text">
             <string>Fuse TV</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="TechCheck">
            <property name="text">
             <string>Tech</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PodcastingCheck">
            <property name="text">
             <string>Podcasting</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="EventsCheck">
            <property name="text">
             <string>Events</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="MarketingCheck">
            <property name="text">
             <string>Marketing</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="DJsCheck">
            <property name="text">
             <string>Live DJs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="MusicCheck">
            <property name="text">
             <string>Music</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="NewsCheck">
            <property name="text">
             <string>News</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PresentersCheck">
            <property name="text">
             <string>Presenters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="NewMusicCheck">
            <property name="text">
             <string>New Music</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Merchandise</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="WristBandCheck">
          <property name="text">
           <string>Wrist Band (£1)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="TShirtCheck">
          <property name="text">
           <string>T-shirt (£6)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="MembershipCheck">
          <property name="text">
           <string>Membership (£5)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="FusePackCheck">
          <property name="text">
           <string>Fuse Pack (£10)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TotalLabel">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Total: £0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="OKButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ClearButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
